packages/xo-web/src/xo-app/sr/index.js

```diff
@@ -53,9 +53,7 @@ import TabXosan from './tab-xosan'
 
   const getVdis = createGetObjectsOfType('VDI').pick(getVdiIds).sort()
   const getVdiSnapshots = createGetObjectsOfType('VDI-snapshot').pick(getVdiIds).sort()
-  const getUnmanagedVdis = createGetObjectsOfType('VDI-unmanaged')
-    .pick(createSelector(getSr, sr => sr.VDIs))
-    .sort()
```

Comment:
Make sure this doesn't break any behaviors. Like there, it shouldn't need to be `keyBy`ed anymore: https://github.com/vatesfr/xen-orchestra/blob/096f443b56bc628ccd325b3e29adcd9899cce08e/packages/xo-web/src/xo-app/sr/tab-general.js#L143

Reply:
@MathieuRA  Did you check this?

Reply:
@Rajaa-BARHTAOUI Yes.